.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -79,20 +79,23 @@ jobs:
     outputs:
       tested: ${{ steps.tested-tree.outputs.src }}
     steps:
-      - name: Retrieve pull request's base and head
-        if: ${{ github.event.issue.pull_request }}
-        id: pr
-        run: |
-          curl ${{ github.event.issue.pull_request.url }} > pr.json
-          echo "::set-output name=base::$(jq -r '.base.sha' pr.json)"
-          echo "::set-output name=head::$(jq -r '.head.sha' pr.json)"
       # Because we run on issue comments, we need to checkout the code for
       # paths-filter to work.
       - name: Checkout code
         if: ${{ github.event.issue.pull_request }}
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
         with:
           persist-credentials: false
+      - name: Retrieve pull request's base and head
+        if: ${{ github.event.issue.pull_request }}
+        id: pr
+        run: |
+          curl ${{ github.event.pull_request.commits_url }} > commits.json
```

Comment:
I think it would be better if we followed GitHub's recommendations of passing in the token as header to the `curl` request, like we do in the `installation-and-connectivity` job's `vars` step. Two reasons:
- It "incidentally works" without it because the repo is public, but I feel it'd be better to make it proper.
- [Unauthenticated requests are rate limited](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api#authentication).

Reply:
Thanks! I fixed it for this `curl` and the existing one.